src/Panel/BasicPanel.php

```diff
@@ -12,20 +12,41 @@
 
 final class BasicPanel implements Panel
 {
+	private string $defaultLocale = 'en';
+
+	private ?Localization $localization = null;
+
+
 	public function __construct(
-		private Localization $localization,
```

Comment:
Tady to mělo zůstat, ale jako nullable:

```php
private ?Localization $localization = null,
```

Reply:
Vytváři chybu v analýze Codacy Static Code Analysis 
Avoid unused parameters on PHP 8: Constructor property

Reply:
Upraveno, chyba znovu zde:
https://app.codacy.com/gh/baraja-core/admin-bar/pullRequest?prid=9106095